highlight/events-explainer.md

```diff
@@ -0,0 +1,250 @@
+# Events for Highlighted Ranges
```

Comment:
Please mention in the PR description that this is an old explainer you are resurrecting for reference purposes, not something new